docker/demo/config/test-suite/multi-writer-1.properties

```diff
@@ -35,7 +35,7 @@ hoodie.datasource.write.recordkey.field=_row_key
 hoodie.datasource.write.keygenerator.class=org.apache.hudi.keygen.TimestampBasedKeyGenerator
 hoodie.datasource.write.partitionpath.field=timestamp
 
-hoodie.write.concurrency.mode=optimistic_concurrency_control
+hoodie.write.concurrency.mode=OPTIMISTIC_CONCURRENCY_CONTROL
```

Comment:
Is upper-case config value required?  Can you still ensure the lower-case value `optimistic_concurrency_control` works as before?

Reply:
Addressed